markdown/core.py

```diff
@@ -191,8 +227,14 @@ def reset(self):
 
         return self
 
-    def set_output_format(self, format):
-        """ Set the output format for the class instance. """
+    def set_output_format(self, format: str) -> Markdown:
+        """
+        Set the output format for the class instance.
+
+        Arguments:
+            format (str): Must be a known value in `Markdown.output_formats`.
```

Comment:
`(str)` not necessary here since the parameter is already type annotated in the signature.

Reply:
Thanks. I figured that out at some point and then I thought I removed them all. Guess I missed this one.